pkgs/applications/misc/fehlstart/patch

```diff
@@ -0,0 +1,13 @@
+--- fehlstart-9f4342d75ec5e2a46c13c99c34894bc275798441-src/fehlstart.c	1970-01-01 01:00:01.000000000 +0100
++++ fehlstart_nixos.c	2016-08-04 22:57:40.831680431 +0200
+@@ -77,8 +77,8 @@
+ #define INPUT_STRING_SIZE       20
+ #define SHOW_IGNORE_TIME        100000
+ #define PI                      (0x1.921fb54442d18p+1)
+-#define APPLICATIONS_DIR_0      "/usr/share/applications"
+-#define APPLICATIONS_DIR_1      "/usr/local/share/applications"
++#define APPLICATIONS_DIR_0      ".nix-profile/share/applications"
```

Comment:
@the-kenny I should patch it so that it would read the content of the NIX_PROFILES environment variable?